faq.rst

```diff
@@ -109,23 +109,25 @@ How can I secure the access to my web application?
 OpenShift supports secure routes and everything is prepared on APPUiO to have
 it secured easily. Just edit the route and change the termination type to ``edge``.
 There is a default trusted certificate in place for ``*.appuioapp.ch`` which is
-used in this case. If you want to use your own certificate, see `Routes <https://docs.openshift.com/enterprise/latest/dev_guide/routes.html>`__.
+used in this case. If you want to use your own certificate, see `Routes
+<https://docs.openshift.com/enterprise/latest/dev_guide/routes.html>`__.
 
 Can I run a database on APPUiO?
 -------------------------------
 
-We provide shared persistent storage using GlusterFS. Please make sure that the database intended to use is capable
-of storing it's data on a shared filesystem. We don't recommend running production databases with GlusterFS as
-storage backend. If you don't fear it, we strongly urge you to at least have a regular database dump available.
-On ``https://github.com/appuio?q=backup`` you can find some database dump applications.
-For highly-available and high-performance managed databases, please contact the APPUiO team under `hello`_.
+Short answer: Yes. But you have to be vigilant. Use the ``gluster-database``
```

Comment:
I would like to have the statement that we don't recommend running production databases on GlusterFS in the short answer, and not just a statement like "Yes. But you have to be vigilant", since we know that there's still issues from time to time even for DBs running on volumes with the `gluster-database` storage class

Reply:
See also the comment of Amar Tumballi https://twitter.com/tumballi/status/1237080752317026306, why this is not recommended.